docs/user/filter_expressions.rst

```diff
@@ -222,6 +222,21 @@ filter expressions.
 
    Boolean specifying whether the user has enabled Do Not Track.
 
+.. js:attribute:: normandy.recipe
+
+   Object containing information about the recipe being checked. Only documented
+   attributes are guaranteed to be available.
+
+   .. js:attribute:: normandy.recipe.id
```

Comment:
So, looking at the test, we don't *only* include this and `arguments`... from the perspective of "I can accurately predict what will happen when you load this recipe", it would make more sense to me if we only included the properties we "needed" and/or if those matched what was documented... is that difficult to do here?